configs/ote/custom-object-detection/cspdarknet_YOLOX/template.yaml

```diff
@@ -0,0 +1,53 @@
+# Description.
+model_template_id: Custom_Object_Detection_YOLOX
+name: YOLOX
+task_type: DETECTION
+task_family: VISION
+instantiation: "CLASS"
+summary: YOLOX with CSP-Darknet backbone.
```

Comment:
Please take a look at summaries of other models and put something more meaningful for anyone who is not a savvy data scientist.

Reply:
done